spacy/cli/project/run.py

```diff
@@ -71,6 +74,12 @@ def project_run(
     commands = {cmd["name"]: cmd for cmd in config.get("commands", [])}
     workflows = config.get("workflows", {})
     validate_subcommand(list(commands.keys()), list(workflows.keys()), subcommand)
+
+    req_path = project_dir / "requirements.txt"
+    if config.get("check_requirements", True) and os.path.exists(req_path):
```

Comment:
Do we want to have this setting in the root of the yml file - i.e. right below `title` and `description`? I guess it's a better spot than in the `vars` section, which is more for user-defined variables?

Reply:
I don't think there's anywhere else that makes sense other than the top level.